main.go

```diff
@@ -663,8 +740,14 @@ func configureSite(ctx *cli.Context, siteNum int, siteStrs []string, healthCheck
 
 		proxy := &httputil.ReverseProxy{
 			Director: func(r *http.Request) {
-				r.Header.Add("X-Forwarded-Host", r.Host)
-				r.Header.Add("X-Real-IP", r.RemoteAddr)
+				if strings.HasPrefix(r.RemoteAddr, "localhost") || strings.HasPrefix(r.RemoteAddr, "127.0.0.1") {
+					r.Header.Add("X-Forwarded-Host", getHost(r))
+					r.Header.Add("X-Real-IP", getRemoteAddr(r))
+					r.Header.Add("X-Forwarded-For", globalClientIP)
+				} else {
+					r.Header.Add("X-Forwarded-Host", r.Host)
+					r.Header.Add("X-Real-IP", r.RemoteAddr)
```

Comment:
Btw this is actually an existing bug, it must not send port information in x-real-ip @shtripat

Reply:
Oh ok. Will remove the port information from value.